default.nix

```diff
@@ -86,6 +86,9 @@ let
       pkgs.mkShell {
         DATA_CACHE_DIRECTORY = toString ./. + "/.data_cache";
         REDIS_SOCKET_URL = "unix:///run/redis/redis.sock";
+        # `./src/website/tracker/settings.py` by default looks for LOCAL_NIXPKGS_CHECKOUT
+        # in the root of the repo. Make it the default here for local development.
+        LOCAL_NIXPKGS_CHECKOUT = toString ./. + "/nixpkgs";
```

Comment:
Hm, this seems a bit weird, by default, we should probably reuse global defaults or something.

Reply:
Or maybe it should end up in some `.state` directory, not sure.

Reply:
Or a cache directory like `DATA_CACHE_DIRECTORY`